setup.py

```diff
@@ -33,6 +33,7 @@
     'numpy>=1.16',
     'pandas>=0.24',
     'matplotlib>=3.0',
+    'packaging>=29.0',
```

Comment:
Why this version floor?

Reply:
(I am mostly asking if you have a link/reference to support the choice ... I don't know the right answer here).

Reply:
I was testing with the most recent `packaging` version. I am going to make some tests to evaluate if it can be used with previous versions.

Reply:
I would expect the parse behavior to be highly stable across versions of packaging; I'm not sure it will be necessary to specify a minimum version.